core/src/main/java/org/elasticsearch/cluster/metadata/IndexTemplateMetaData.java

```diff
@@ -207,6 +248,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeString(cursor.key);
             cursor.value.writeTo(out);
         }
+        out.writeOptionalVInt(version);
```

Comment:
We can also check for version here so that we are coherent with the `readFrom` method?